src/index.html

```diff
@@ -7,8 +7,58 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Inter&family=Poppins:wght@300;500&family=Roboto:wght@500&display=swap" rel="stylesheet">
+
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo" href="index.html">
+        <img src="images/logo.png" alt="some-logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav-ul">
+          <li class="li-nav"><a class="element is-active" href="#">Apple</a></li>
```

Comment:
```suggestion
        <ul class="nav__list">
          <li class="list__item"><a class="element is-active" href="#">Apple</a></li>
```
good progress - those class names are way more readable, but I think I would name them like that